libkbfs/folder_branch_ops.go

```diff
@@ -3960,6 +3950,36 @@ func (fbo *folderBranchOps) notifyOneOpLocked(ctx context.Context,
 	afterUpdateFn func() error) error {
 	fbo.headLock.AssertLocked(lState)
 
+	// We need to get unlinkPath before calling UpdatePointers so that
+	// nodeCache.Unlink can properly update cachedPath.
+	unlinkPath, toUnlink := func() (path, bool) {
+		var (
```

Comment:
Is there a benefit to using the `()` construct?  Seems like it just uses two extra lines for no reason.